parlai/agents/transformer/modules.py

```diff
@@ -766,23 +771,43 @@ def forward_layers(
             return encoding after applying decoder layers, as well
             as new incremental decoding state.
         """
-        new_incr_state = {}
         if getattr(self.layers, 'is_model_parallel', False):
+            assert (
+                not torch.jit.is_scripting()
+            ), 'model_parallel not currently supported when TorchScripting!'
             tensor, new_incr_state = self._apply_model_parallel(
                 tensor, encoder_output, encoder_mask, incr_state
             )
         else:
+            new_incr_state_by_layer = []
             for idx, layer in enumerate(self.layers):
-                tensor, new_incr_state[idx] = layer(
+                if incr_state is not None:
+                    single_layer_incr_state = {
+                        key: val[..., idx] for key, val in incr_state.items()
```

Comment:
what is the ... syntax? Can you leave a comment explaining this?

Reply:
Yes, adding comment